tests/pathological.c

```diff
@@ -0,0 +1,44 @@
+/* iso_alloc memset_sanity.c
+ * Copyright 2023 - chris.rohlf@gmail.com */
+
+#include "iso_alloc.h"
+#include "iso_alloc_internal.h"
+
+#if !MEMSET_SANITY
+#error "This test intended to be run with -DMEMSET_SANITY=1"
```

Comment:
copy paste artifact? Also up top in the comment.

Reply:
How so? It's part of the sanity part of the tests